chart/compass/charts/gateway/templates/oathkeeper-rules.yaml

```diff
@@ -157,6 +157,10 @@ spec:
 {{ toYaml .Values.global.oathkeeper.mutators.certificateResolverService | indent 4 }}
   - handler: hydrator
 {{ toYaml .Values.global.oathkeeper.mutators.tenantMappingService | indent 4 }}
+  - handler: header
+    config:
+      headers:
+        {{ .Values.global.gateway.headers.rateLimit }}: "{{ .Values.rateLimit.headerValueTemplate }}"
```

Comment:
You need to add the header to `compass-connector-certs` ory rule as well to cover certificate renewal in the Runtime bucket instead of the common bucket.